data/json/furniture_and_terrain/furniture-plumbing.json

```diff
@@ -75,7 +75,7 @@
       "sound": "porcelain breaking!",
       "sound_fail": "whunk!",
       "items": [
-        { "item": "cu_pipe", "prob": 50 },
```

Comment:
This is probably the drainage pipe + syphon, there ought to be a chance for it to be metallic.